### ISSUE TITLE
Version dates of minified plugins don't match dates of source versions

### ISSUE THREAD

**Issue by benebun** [malsup/cycle2#622]
The minified plugins have a different version date from the source versions:
E.g. the carousel plugin:
v20141007 http://malsup.github.io/min/jquery.cycle2.carousel.min.js
vs.
v20130528 http://malsup.github.io/jquery.cycle2.carousel.js
Swipe plugin:
v20141007 http://malsup.github.io/min/jquery.cycle2.swipe.min.js
v20121120 http://malsup.github.io/jquery.cycle2.swipe.js

Is the version mismatch maybe just because of an automatic update of the date at the time of the minification or are the contents actually different?

**Comment by malsup** [comment]
Yes, those dates are generated during manification.  The contents should be
equivalent between the min and non-min versions.
